templates/view-submitted-response.html

```diff
@@ -29,6 +29,17 @@ <h1 class="u-mt-m">
   </h1>
   {{ onsMetadata(content.metadata)}}
   {% if not content.view_submitted_response.expired %}
+    {{
+      onsButton({
+        "type": 'button',
+        "text": 'Print answers',
+        "buttonStyle": "print",
```

Comment:
The button in the prototype is the small version, which can be done via `"classes": "btn--small",`